py3status/modules/spaceapi.py

```diff
@@ -23,12 +25,15 @@
 import datetime
 import json
 import urllib.request
+import shlex
+import subprocess
 
 
 class Py3status:
     """
     """
     # available configuration parameters
+    button_action = 3
```

Comment:
I'm very keen for all these type things to be `button_xxxx` to be consistent across modules.  Historically it's been a bit of a free for all.